rails-time-track/db/seeds.rb

```diff
@@ -5,3 +5,72 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require 'faker'
+
+3.times do
+  User.create(
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    # role: Faker::Job.position,
+    role: "Owner",
+    rate: Faker::Number.between(1000,2000)
+  )
+end
+
+3.times do
+  User.create(
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    role: "Manager",
+    rate: Faker::Number.between(1000,2000)
+  )
+end
+
+4.times do
+  User.create(
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    role: "Analist",
+    rate: Faker::Number.between(1000,2000)
+  )
+end
```

Comment:
Same here, maybe it could be better to manually create 3-4 analyst with specific rates